NitroxPatcher/Patches/Dynamic/GotoConsoleCommand_GotoPosition_Patch.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Reflection;
+using NitroxModel.Helper;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class GotoConsoleCommand_GotoPosition_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((GotoConsoleCommand t) => t.GotoPosition(default, default));
+
+    public static bool Prefix(Vector3 position, bool gotoImmediate = false)
```

Comment:
You don't need to include the parameters if you don't need them.
```suggestion
    public static bool Prefix(Vector3 position)
```